Source/items.h

```diff
@@ -182,65 +182,65 @@ struct Item {
 	 * @brief Contains Information for current Animation
 	 */
 	AnimationInfo AnimInfo;
-	bool _iDelFlag = false; // set when item is flagged for deletion, deprecated in 1.02

```

Comment:
The default values for these members are to help ensure items are default-initialised appropriately, where possible they should be retained.

Reply:
weird... I didn't make any of those changes. Is that a clang format thing?

Reply:
It was a recent addition so more likely it was an improper merge/rebase when you created the branch. clang format is always applied so It wouldn't have been possible to add the defaults in the first place if that was the case.

Reply:
ah ok - I'll fix it